components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.commons/src/main/java/org/wso2/carbon/cluster/coordinator/commons/node/NodeDetail.java

```diff
@@ -66,6 +73,25 @@ public NodeDetail(String nodeId, String groupId, boolean isCoordinator, long las
     }
 
     /**
+     *
+     * @param nodeId        node ID
+     * @param groupId       cluster agent's socket address
+     * @param lastHeartbeat last heartbeat received from the node
+     * @param isCoordinator check if this node is the coordinator
+     * @param isNewNode     true if new node
+     */
+    public NodeDetail(String nodeId, String groupId, boolean isCoordinator, long lastHeartbeat,
+                      boolean isNewNode) {
```

Comment:
isNewNode parameter can be on the line above. Please check the formatting.

Reply:
Done